env_vars_32.sh

```diff
@@ -4,4 +4,6 @@
 set -x
 OPENBLAS_VERSION="v0.3.7"
 MACOSX_DEPLOYMENT_TARGET=10.9
-CFLAGS="-msse2 -std=c99 -fno-strict-aliasing"
+# Causes failure for pre-1.19 in np.reciprocal
+# CFLAGS="-msse2 -std=c99 -fno-strict-aliasing"
+CFLAGS="-std=c99 -fno-strict-aliasing"
```

Comment:
I hope to get 1.18.4 out using the travis/appveyor system, so this is really an azure testing option. Any idea why the test failures don't show up with travis/appveyor? Once we have things working we can start pointing to numpy master branch for numpy-wheels master, sometimes we want to trigger a nightly build unrelated to cron.